### PR TITLE
Update ru.txt

### PR DESCRIPTION
**Pull request by geoffthemedio**
Resolves conflicts of and supercedes #2793

### CODE REVIEW

**Comment by o01eg** [comment]
I remember I left some notes about previous PR. Also it's a still very big PR to be correctly reviewed.

**Comment by geoffthemedio** [comment]
> I remember I left some notes about previous PR. Also it's a still very big PR to be correctly reviewed.

It looked like there were replies to all your comments, with no additional follow up or re-reply for some time...

I'd like to merge this before the next release, but will certainly wait for your approval.

**Comment by Cjkjvfnby** [comment]
> Also it's a still very big PR to be correctly reviewed.

Technically it is easy to split PR to separate PRs because translation entries are not related(almost).  I can help @naiffuR  with it.     

@o01eg what do you think?

File: default/stringtables/ru.txt
Comment (o01eg) [inline default/stringtables/ru.txt:1448]: @naiffuR `когда и когда`

**Review by o01eg — CHANGES_REQUESTED** [review]

**Comment by o01eg** [comment]
@Cjkjvfnby Yes, splitting PR would be nice.

File: default/stringtables/ru.txt
Comment (naiffuR) [reply default/stringtables/ru.txt:1448]: fix: когда и какой игрок отправил приказ хода

**Comment by naiffuR** [comment]
Fix [ru.txt](https://github.com/freeorion/freeorion/files/4562892/ru.txt)

**Comment by geoffthemedio** [comment]
> Fix [ru.txt](https://github.com/freeorion/freeorion/files/4562892/ru.txt)

Not sure what you mean...

**Pull request closed without merging**
